WooCommerce/Classes/ViewRelated/Search/Order/OrderSearchUICommand.swift

```diff
@@ -101,7 +101,11 @@ final class OrderSearchUICommand: SearchUICommand {
 
 private extension OrderSearchUICommand {
     func configureResultsController() {
-        try? statusResultsController.performFetch()
+        do {
+            try statusResultsController.performFetch()
+        } catch {
+            DDLogError("⛔️ Unable to fetch Order search results for Site \(siteID): \(error)")
```

Comment:
nit: I think this results controller is for order statuses?

```suggestion
            DDLogError("⛔️ Unable to fetch Order statuses in order search for Site \(siteID): \(error)")
```